src/test/java/seedu/address/storage/toDoListStorage/XmlAdaptedToDoListEventTest.java

```diff
@@ -0,0 +1,81 @@
+package seedu.address.storage.toDoListStorage;
+
```

Comment:
Just a qn, is there any particular reason for keeping this (and the other ToDoList storage classes) in a separate package?

Reply:
I just want to see it clearly. Can delete the package.